[2024-08-24T20:02:53.135+0000] {processor.py:186} INFO - Started process (PID=555) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:53.136+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:02:53.142+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:02:53.141+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:53.149+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:02:53.146+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 5
    schedule_interval=@once)
                      ^
SyntaxError: invalid syntax
[2024-08-24T20:02:53.150+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:53.202+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.083 seconds
[2024-08-24T20:02:57.167+0000] {processor.py:186} INFO - Started process (PID=556) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:57.168+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:02:57.172+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:02:57.171+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:57.179+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:02:57.176+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 5
    schedule_interval="@once)
                      ^
SyntaxError: unterminated string literal (detected at line 5)
[2024-08-24T20:02:57.181+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:57.213+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.061 seconds
[2024-08-24T20:02:58.178+0000] {processor.py:186} INFO - Started process (PID=557) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:58.179+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:02:58.187+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:02:58.183+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:58.193+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:02:58.191+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 5
    schedule_interval="@once2)
                      ^
SyntaxError: unterminated string literal (detected at line 5)
[2024-08-24T20:02:58.194+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:02:58.248+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.085 seconds
[2024-08-24T20:03:00.285+0000] {processor.py:186} INFO - Started process (PID=558) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:00.286+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:00.293+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:00.292+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:00.299+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:00.297+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    )
     ^
SyntaxError: expected ':'
[2024-08-24T20:03:00.300+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:00.328+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.056 seconds
[2024-08-24T20:03:03.346+0000] {processor.py:186} INFO - Started process (PID=559) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:03.347+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:03.353+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:03.352+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:03.358+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:03.356+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start)
         ^
SyntaxError: positional argument follows keyword argument
[2024-08-24T20:03:03.359+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:03.394+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.060 seconds
[2024-08-24T20:03:05.414+0000] {processor.py:186} INFO - Started process (PID=565) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:05.415+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:05.418+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:05.418+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:05.423+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:05.421+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=)
    ^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-08-24T20:03:05.424+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:05.452+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.049 seconds
[2024-08-24T20:03:10.504+0000] {processor.py:186} INFO - Started process (PID=566) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:10.506+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:10.512+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:10.511+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:10.520+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:10.517+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=datetime)
                        ^
SyntaxError: expected ':'
[2024-08-24T20:03:10.521+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:10.568+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.082 seconds
[2024-08-24T20:03:12.566+0000] {processor.py:186} INFO - Started process (PID=567) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:12.567+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:12.575+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:12.574+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:12.582+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:12.579+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=datetime(20))
                            ^
SyntaxError: expected ':'
[2024-08-24T20:03:12.584+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:12.707+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.155 seconds
[2024-08-24T20:03:16.690+0000] {processor.py:186} INFO - Started process (PID=568) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:17.065+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:17.071+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:17.070+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:17.077+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:17.075+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=datetime(2022, 8,1))
                                   ^
SyntaxError: expected ':'
[2024-08-24T20:03:17.079+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:17.112+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.436 seconds
[2024-08-24T20:03:17.843+0000] {processor.py:186} INFO - Started process (PID=569) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:17.844+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:17.847+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:17.847+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:17.852+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:17.850+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=datetime(2022, 8, 1))
                                    ^
SyntaxError: expected ':'
[2024-08-24T20:03:17.853+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:17.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.065 seconds
[2024-08-24T20:03:19.830+0000] {processor.py:186} INFO - Started process (PID=570) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:19.831+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:19.836+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:19.835+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:19.840+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:19.839+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=datetime(2024, 8, 1))
                                    ^
SyntaxError: expected ':'
[2024-08-24T20:03:19.841+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:19.867+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.049 seconds
[2024-08-24T20:03:25.972+0000] {processor.py:186} INFO - Started process (PID=572) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:25.974+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:25.980+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:25.979+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:25.985+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:25.983+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 6
    start_date=datetime(2024, 8, 1) as dag)
                                    ^^
SyntaxError: invalid syntax
[2024-08-24T20:03:25.986+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:26.027+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.074 seconds
[2024-08-24T20:03:40.090+0000] {processor.py:186} INFO - Started process (PID=578) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:40.091+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:40.098+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:40.095+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:40.104+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:40.102+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 8
    start_date=datetime(2024, 8, 1) as dag)
                                    ^^
SyntaxError: invalid syntax
[2024-08-24T20:03:40.105+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:40.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.064 seconds
[2024-08-24T20:03:47.238+0000] {processor.py:186} INFO - Started process (PID=579) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:47.240+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:03:47.247+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:47.246+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:47.254+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:03:47.251+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 2
    from 
         ^
SyntaxError: invalid syntax
[2024-08-24T20:03:47.255+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:03:47.294+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.078 seconds
[2024-08-24T20:04:11.553+0000] {processor.py:186} INFO - Started process (PID=587) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:11.564+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:04:11.570+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:11.569+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:11.575+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:11.573+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 8
    start_date=datetime(2024, 8, 1) as dag)
                                    ^^
SyntaxError: invalid syntax
[2024-08-24T20:04:11.576+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:11.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.069 seconds
[2024-08-24T20:04:23.652+0000] {processor.py:186} INFO - Started process (PID=588) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:23.654+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:04:23.658+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:23.657+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:23.662+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:23.661+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 8
    start_date=datetime(2024, 8, 1)) as dsag:
                                             ^
IndentationError: expected an indented block after 'with' statement on line 5
[2024-08-24T20:04:23.663+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:23.693+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.050 seconds
[2024-08-24T20:04:25.750+0000] {processor.py:186} INFO - Started process (PID=590) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:25.751+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:04:25.757+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:25.757+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:25.763+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:25.761+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 9
    
    ^
IndentationError: expected an indented block after 'with' statement on line 5
[2024-08-24T20:04:25.764+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:25.798+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.061 seconds
[2024-08-24T20:04:29.820+0000] {processor.py:186} INFO - Started process (PID=591) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:29.821+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:04:29.826+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:29.825+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:29.841+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:29.834+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = 
         ^
SyntaxError: invalid syntax
[2024-08-24T20:04:29.843+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:29.918+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.127 seconds
[2024-08-24T20:04:34.974+0000] {processor.py:186} INFO - Started process (PID=598) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:34.977+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:04:34.984+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:34.983+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:35.006+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:34.989+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 2, in <module>
    from airflow.operators import PythonOperator
ImportError: cannot import name 'PythonOperator' from 'airflow.operators' (/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/__init__.py)
[2024-08-24T20:04:35.013+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:35.076+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.119 seconds
[2024-08-24T20:04:45.243+0000] {processor.py:186} INFO - Started process (PID=599) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:45.245+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:04:45.248+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.248+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:45.262+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:04:45.817+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.815+0000] {override.py:1858} INFO - Created Permission View: can edit on DAG:pythonoperator
[2024-08-24T20:04:45.846+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.845+0000] {override.py:1858} INFO - Created Permission View: can read on DAG:pythonoperator
[2024-08-24T20:04:45.856+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.855+0000] {override.py:1858} INFO - Created Permission View: can delete on DAG:pythonoperator
[2024-08-24T20:04:45.856+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.856+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:04:45.889+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.889+0000] {dag.py:3234} INFO - Creating ORM DAG for pythonoperator
[2024-08-24T20:04:45.912+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:04:45.911+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to 2024-08-01 00:00:00+00:00, run_after=2024-08-01 00:00:00+00:00
[2024-08-24T20:04:45.938+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.706 seconds
[2024-08-24T20:05:14.477+0000] {processor.py:186} INFO - Started process (PID=607) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:14.489+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:14.493+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:14.492+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:14.499+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:14.497+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 2, in <module>
    from airflow.operators.python import P
ImportError: cannot import name 'P' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py)
[2024-08-24T20:05:14.500+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:14.532+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.066 seconds
[2024-08-24T20:05:15.491+0000] {processor.py:186} INFO - Started process (PID=608) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:15.492+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:15.496+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:15.495+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:15.509+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:15.536+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:15.535+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:05:15.589+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:15.589+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to 2024-08-01 00:00:00+00:00, run_after=2024-08-01 00:00:00+00:00
[2024-08-24T20:05:15.623+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.141 seconds
[2024-08-24T20:05:24.679+0000] {processor.py:186} INFO - Started process (PID=609) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:24.681+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:24.693+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:24.692+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:24.709+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:24.701+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10, in <module>
    t2 = PythonOperator(task_id="")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-08-24T20:05:24.711+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:24.747+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.087 seconds
[2024-08-24T20:05:26.852+0000] {processor.py:186} INFO - Started process (PID=610) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:26.857+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:26.863+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:26.862+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:26.952+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:26.943+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10, in <module>
    t2 = PythonOperator(task_id=""
         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'python_callable'
[2024-08-24T20:05:26.954+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:27.066+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.234 seconds
[2024-08-24T20:05:29.199+0000] {processor.py:186} INFO - Started process (PID=612) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:29.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:29.206+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:29.205+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:29.212+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:29.209+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id=""
                                ^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:05:29.213+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:29.255+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.067 seconds
[2024-08-24T20:05:31.196+0000] {processor.py:186} INFO - Started process (PID=613) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:31.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:31.203+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:31.202+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:31.208+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:31.207+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id=""
                                ^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:05:31.211+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:31.243+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.060 seconds
[2024-08-24T20:05:39.262+0000] {processor.py:186} INFO - Started process (PID=620) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:39.264+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:39.270+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:39.269+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:39.277+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:39.275+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id="python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:05:39.279+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:39.318+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.068 seconds
[2024-08-24T20:05:42.360+0000] {processor.py:186} INFO - Started process (PID=621) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:42.361+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:42.372+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:42.371+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:42.388+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:42.386+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id="pyt"
                                ^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:05:42.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:42.478+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.158 seconds
[2024-08-24T20:05:44.484+0000] {processor.py:186} INFO - Started process (PID=622) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:44.489+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:44.504+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:44.503+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:44.513+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:44.509+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id="
                                ^
SyntaxError: unterminated string literal (detected at line 10)
[2024-08-24T20:05:44.515+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:44.584+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.130 seconds
[2024-08-24T20:05:48.693+0000] {processor.py:186} INFO - Started process (PID=623) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:48.694+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:48.707+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:48.706+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:48.726+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:48.719+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id=""
                                ^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:05:48.728+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:48.814+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.176 seconds
[2024-08-24T20:05:55.783+0000] {processor.py:186} INFO - Started process (PID=624) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:55.788+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:05:55.794+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:55.793+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:55.803+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:05:55.800+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 10
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:05:55.804+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:05:55.867+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.102 seconds
[2024-08-24T20:06:05.016+0000] {processor.py:186} INFO - Started process (PID=632) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:05.017+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:05.024+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:05.023+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:05.031+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:05.028+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 12
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:05.032+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:05.087+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.090 seconds
[2024-08-24T20:06:22.173+0000] {processor.py:186} INFO - Started process (PID=633) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:22.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:22.210+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:22.209+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:22.219+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:22.216+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 13
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:22.220+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:22.278+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.128 seconds
[2024-08-24T20:06:25.330+0000] {processor.py:186} INFO - Started process (PID=634) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:25.331+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:25.339+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:25.339+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:25.346+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:25.343+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 13
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:25.347+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:25.385+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.070 seconds
[2024-08-24T20:06:36.764+0000] {processor.py:186} INFO - Started process (PID=642) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:36.766+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:36.773+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:36.772+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:36.785+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:36.781+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 13
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:36.787+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:36.860+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.120 seconds
[2024-08-24T20:06:42.586+0000] {processor.py:186} INFO - Started process (PID=643) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:42.587+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:42.592+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:42.591+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:42.598+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:42.596+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 5
    //
    ^^
SyntaxError: invalid syntax
[2024-08-24T20:06:42.599+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:42.631+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.057 seconds
[2024-08-24T20:06:44.606+0000] {processor.py:186} INFO - Started process (PID=644) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:44.607+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:44.611+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:44.611+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:44.616+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:44.614+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:44.617+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:44.658+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.062 seconds
[2024-08-24T20:06:48.726+0000] {processor.py:186} INFO - Started process (PID=645) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:48.727+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:48.731+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:48.731+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:48.736+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:48.735+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:48.737+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:48.768+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.056 seconds
[2024-08-24T20:06:53.790+0000] {processor.py:186} INFO - Started process (PID=646) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:53.793+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:53.796+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:53.796+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:53.801+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:53.799+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:53.802+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:53.831+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.053 seconds
[2024-08-24T20:06:55.863+0000] {processor.py:186} INFO - Started process (PID=647) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:55.865+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:06:55.869+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:55.868+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:55.873+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:06:55.872+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:06:55.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:06:55.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.076 seconds
[2024-08-24T20:07:23.241+0000] {processor.py:186} INFO - Started process (PID=654) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:23.242+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:07:23.245+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:23.244+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:23.251+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:23.249+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:07:23.252+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:23.277+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.048 seconds
[2024-08-24T20:07:27.285+0000] {processor.py:186} INFO - Started process (PID=655) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:27.287+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:07:27.289+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:27.289+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:27.296+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:27.294+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:07:27.298+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:27.334+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.062 seconds
[2024-08-24T20:07:31.968+0000] {processor.py:186} INFO - Started process (PID=657) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:31.970+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:07:31.972+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:31.972+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:31.979+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:31.977+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:07:31.981+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:32.040+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.085 seconds
[2024-08-24T20:07:37.014+0000] {processor.py:186} INFO - Started process (PID=664) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:37.015+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:07:37.018+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:37.017+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:37.023+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:07:37.021+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:07:37.024+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:07:37.059+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.058 seconds
[2024-08-24T20:08:07.556+0000] {processor.py:186} INFO - Started process (PID=672) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:07.570+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:08:07.572+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:07.572+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:07.581+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:07.578+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:08:07.582+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:07.616+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.071 seconds
[2024-08-24T20:08:18.642+0000] {processor.py:186} INFO - Started process (PID=673) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:18.643+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:08:18.646+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:18.644+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:18.652+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:18.649+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14
    t2 = PythonOperator(task_id="hello_with_python"
                                ^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-08-24T20:08:18.654+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:18.689+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.059 seconds
[2024-08-24T20:08:21.818+0000] {processor.py:186} INFO - Started process (PID=674) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:21.820+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:08:21.822+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:21.822+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:21.868+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:21.862+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 14, in <module>
    t2 = PythonOperator(task_id="hello_with_python",
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 224, in __init__
    raise AirflowException("`python_callable` param must be callable")
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2024-08-24T20:08:21.870+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:21.911+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.106 seconds
[2024-08-24T20:08:45.947+0000] {processor.py:186} INFO - Started process (PID=681) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:45.952+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:08:45.954+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:45.953+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:45.961+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:45.958+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/2-pythonoperator.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/2-pythonoperator.py", line 15
    python_callable=print_hello")
                               ^
SyntaxError: unterminated string literal (detected at line 15)
[2024-08-24T20:08:45.963+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:46.002+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.068 seconds
[2024-08-24T20:08:47.975+0000] {processor.py:186} INFO - Started process (PID=682) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:47.977+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:08:47.980+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:47.979+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:48.001+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:08:48.299+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:48.299+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:08:48.354+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:08:48.353+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to 2024-08-01 00:00:00+00:00, run_after=2024-08-01 00:00:00+00:00
[2024-08-24T20:08:48.398+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.440 seconds
[2024-08-24T20:09:18.700+0000] {processor.py:186} INFO - Started process (PID=690) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:18.702+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:09:18.704+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:18.704+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:18.718+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:18.760+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:18.760+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:09:18.809+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:18.808+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:09:18.841+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.153 seconds
[2024-08-24T20:09:30.833+0000] {processor.py:186} INFO - Started process (PID=691) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:30.838+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:09:30.841+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:30.840+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:30.866+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:30.935+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:30.934+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:09:31.007+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:31.006+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:09:31.068+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.258 seconds
[2024-08-24T20:09:32.006+0000] {processor.py:186} INFO - Started process (PID=692) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:32.007+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:09:32.010+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:32.009+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:32.042+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:09:32.121+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:32.121+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:09:32.194+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:09:32.194+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:09:32.250+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.259 seconds
[2024-08-24T20:10:02.576+0000] {processor.py:186} INFO - Started process (PID=701) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:10:02.727+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:10:02.730+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:10:02.729+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:10:02.750+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:10:02.810+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:10:02.810+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:10:02.866+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:10:02.865+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:10:02.905+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.342 seconds
[2024-08-24T20:10:33.197+0000] {processor.py:186} INFO - Started process (PID=709) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:10:33.218+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:10:33.220+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:10:33.220+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:10:33.239+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:10:33.295+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:10:33.295+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:10:33.350+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:10:33.350+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:10:33.386+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.203 seconds
[2024-08-24T20:11:03.675+0000] {processor.py:186} INFO - Started process (PID=717) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:11:03.676+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:11:03.680+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:11:03.679+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:11:03.698+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:11:03.761+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:11:03.760+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:11:03.827+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:11:03.827+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:11:03.872+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.213 seconds
[2024-08-24T20:11:34.160+0000] {processor.py:186} INFO - Started process (PID=725) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:11:34.161+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:11:34.163+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:11:34.162+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:11:34.178+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:11:34.219+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:11:34.219+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:11:34.262+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:11:34.261+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:11:34.294+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.144 seconds
[2024-08-24T20:12:04.665+0000] {processor.py:186} INFO - Started process (PID=732) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:12:04.780+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:12:04.789+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:04.784+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:12:04.913+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:12:05.127+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:05.126+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:12:05.308+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:05.308+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:12:05.390+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.774 seconds
[2024-08-24T20:12:35.973+0000] {processor.py:186} INFO - Started process (PID=740) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:12:35.994+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:12:36.002+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:35.999+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:12:36.027+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:12:36.104+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:36.103+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:12:36.195+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:12:36.194+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:12:36.245+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.307 seconds
[2024-08-24T20:13:06.778+0000] {processor.py:186} INFO - Started process (PID=751) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:13:06.780+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:13:06.782+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:06.782+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:13:06.799+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:13:06.861+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:06.860+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:13:06.921+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:06.921+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:13:06.957+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.195 seconds
[2024-08-24T20:13:37.459+0000] {processor.py:186} INFO - Started process (PID=767) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:13:37.476+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:13:37.479+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:37.478+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:13:37.506+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:13:37.602+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:37.602+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:13:37.675+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:13:37.675+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:13:37.734+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.295 seconds
[2024-08-24T20:14:08.238+0000] {processor.py:186} INFO - Started process (PID=779) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:14:08.241+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:14:08.244+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:08.243+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:14:08.267+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:14:08.335+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:08.334+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:14:08.403+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:08.403+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:14:08.466+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.246 seconds
[2024-08-24T20:14:39.080+0000] {processor.py:186} INFO - Started process (PID=788) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:14:39.081+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:14:39.095+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:39.084+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:14:39.134+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:14:39.389+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:39.388+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:14:39.556+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:14:39.555+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:14:39.666+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.636 seconds
[2024-08-24T20:15:09.995+0000] {processor.py:186} INFO - Started process (PID=796) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:15:10.025+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:15:10.028+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:10.027+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:15:10.043+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:15:10.116+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:10.115+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:15:10.188+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:10.187+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:15:10.235+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.254 seconds
[2024-08-24T20:15:40.558+0000] {processor.py:186} INFO - Started process (PID=804) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:15:40.561+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:15:40.564+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:40.564+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:15:40.601+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:15:40.698+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:40.698+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:15:40.759+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:15:40.758+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:15:40.816+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.278 seconds
[2024-08-24T20:16:11.201+0000] {processor.py:186} INFO - Started process (PID=812) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:16:11.219+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:16:11.222+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:11.221+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:16:11.250+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:16:11.355+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:11.355+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:16:11.460+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:11.460+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:16:11.540+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.355 seconds
[2024-08-24T20:16:41.929+0000] {processor.py:186} INFO - Started process (PID=820) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:16:41.943+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:16:41.946+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:41.945+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:16:41.968+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:16:42.041+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:42.041+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:16:42.098+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:16:42.098+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:16:42.141+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.225 seconds
[2024-08-24T20:17:12.472+0000] {processor.py:186} INFO - Started process (PID=829) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:17:12.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:17:12.476+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:12.476+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:17:12.495+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:17:12.551+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:12.550+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:17:12.598+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:12.597+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:17:12.629+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.171 seconds
[2024-08-24T20:17:43.020+0000] {processor.py:186} INFO - Started process (PID=838) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:17:43.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:17:43.054+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:43.054+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:17:43.080+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:17:43.165+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:43.164+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:17:43.360+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:17:43.359+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:17:43.492+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.527 seconds
[2024-08-24T20:18:13.839+0000] {processor.py:186} INFO - Started process (PID=848) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:18:13.841+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:18:13.844+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:13.843+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:18:13.864+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:18:13.944+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:13.943+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:18:14.007+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:14.007+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:18:14.044+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.220 seconds
[2024-08-24T20:18:44.350+0000] {processor.py:186} INFO - Started process (PID=857) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:18:44.373+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:18:44.379+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:44.375+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:18:44.398+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:18:44.448+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:44.447+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:18:44.508+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:18:44.507+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:18:44.542+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.204 seconds
[2024-08-24T20:19:14.889+0000] {processor.py:186} INFO - Started process (PID=866) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:19:14.891+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:19:14.899+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:14.893+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:19:14.932+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:19:15.055+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:15.054+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:19:15.154+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:15.153+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:19:15.207+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.340 seconds
[2024-08-24T20:19:45.526+0000] {processor.py:186} INFO - Started process (PID=875) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:19:45.544+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:19:45.548+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:45.547+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:19:45.573+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:19:45.693+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:45.692+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:19:45.786+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:19:45.786+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:19:45.844+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.336 seconds
[2024-08-24T20:20:16.138+0000] {processor.py:186} INFO - Started process (PID=884) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:20:16.139+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:20:16.142+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:16.142+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:20:16.157+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:20:16.211+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:16.210+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:20:16.275+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:16.274+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:20:16.320+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.194 seconds
[2024-08-24T20:20:46.759+0000] {processor.py:186} INFO - Started process (PID=894) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:20:46.781+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:20:46.784+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:46.783+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:20:46.797+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:20:46.854+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:46.853+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:20:46.916+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:20:46.915+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:20:46.953+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.207 seconds
[2024-08-24T20:21:17.254+0000] {processor.py:186} INFO - Started process (PID=903) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:21:17.272+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:21:17.275+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:17.274+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:21:17.292+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:21:17.353+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:17.353+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:21:17.411+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:17.410+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:21:17.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.208 seconds
[2024-08-24T20:21:47.780+0000] {processor.py:186} INFO - Started process (PID=912) to work on /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:21:47.797+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/2-pythonoperator.py for tasks to queue
[2024-08-24T20:21:47.799+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:47.799+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:21:47.816+0000] {processor.py:925} INFO - DAG(s) 'pythonoperator' retrieved from /opt/airflow/dags/2-pythonoperator.py
[2024-08-24T20:21:47.867+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:47.866+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-08-24T20:21:47.923+0000] {logging_mixin.py:190} INFO - [2024-08-24T20:21:47.922+0000] {dag.py:4138} INFO - Setting next_dagrun for pythonoperator to None, run_after=None
[2024-08-24T20:21:47.952+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/2-pythonoperator.py took 0.184 seconds
